src/Sylius/Bundle/AttributeBundle/Resources/views/attributeValueForms.html.twig

```diff
@@ -0,0 +1,40 @@
+{% for attributeId, form in forms %}
+    <div data-form-collection="item"
+         data-form-collection-index="{{ count }}"
+         class="collection-item">
+        <div class="collection-box">
+
+            <p class="text-right">
+                <a href="#" data-form-collection="delete" class="btn btn-danger">
+                    <i class="glyphicon glyphicon-remove collection-button-remove"></i>
+                    {{ 'sylius.delete'|trans }}
+                </a>
+            </p>
+
+            <div class="form-group">
+                {% set id = form.vars.label|replace(' ', '_')|lower %}
```

Comment:
form.id or form.name? Don't remember which one would return correct but that looks like a weird way to get form id.